### ISSUE TITLE
Lacking Prefab File - maybe moved?

### ISSUE THREAD

**Issue by norm-ideal** [Okuu-Sama/Tama-Project-2021#14]
https://github.com/Okuu-Sama/Tama-Project-2021/blob/develop/Bardventure/Assets/Nicolas/RhythmCore.cs#L89

No file found at `Assets/Marine/SimpleNote.prefab`.

**Comment by Okuu-Sama** [comment]
Yes I forgot that marine made some change to her code and thus I had to change my code when creating the display.
I will push the new version soon.
